### DISCUSSION TITLE
Mehrere Räume reinigen / MultiRoom cleaning

### DISCUSSION THREAD

**Discussion by kosmix1980** [iobroker-community-adapters/ioBroker.mihome-vacuum#910] [Q&A] [answered]
Hallo zusammen,

ich nutze seit langer Zeit diesen Adapter und bin auch absolut zufrieden und dankbar für eure / deine Arbeit.

Ich habe mir allerdings schon häufiger die Frage gestellt, ob ich etwas falsch mache und habe es heute geschafft diesen Frage hier zu formulieren ;-)

Ich habe mehrere Räume, die ich nacheinander reinigen lasse. Diese Räume haben ggf. unterschiedliche Konfigurationen (Saugleistung, Häufigkeit der Reinigungsdurchläufe etc.). Bisher nutze ich die Variante, dass ich in den Objekten unter jedem Raum den ich reinigen möchte den Datenpunkt "roomClean" auf true setze.

Dann durchläuft der Roboter auch alle Räume, die ich entsprechend auf true gesetzt habe, er macht aber immer nachdem er mit einem Raum fertig ist sich kurz auf den Weg zur Ladestation. Dann scheint er die Info zu bekommen, welchen Raum er als nächstes zu reinigen hat und fährt dann in diesen.
Auch in der MAP sieht das so aus, als wären es einzelne Reinigungsaufträge, da immer nur der aktive Raum farblich gekennzeichnet (und mit der bereits gefahrenen Route dargestellt) wird.

In der Original-App werden alle Räume und die komplette Route die der Roboter gefahren ist dargestellt.

Daher meine Frage, nutze ich die falsche Funktion ?

Ich freue mich auf eure Antworten.

---------------------------------------------------------------------------------

Hello everyone,

I have been using this adapter for a long time and am absolutely satisfied and grateful for your work.

However, I have often asked myself whether I am doing something wrong and today I managed to formulate this question here ;-)

I have several rooms that I have cleaned one after the other (see screenshot). These rooms may have different configurations (suction power, frequency of cleaning cycles, etc.). So far I have used the variant where I set the data point "roomClean" to true in the objects under each room that I want to clean.

The robot then goes through all the rooms that I have set to true, but after it has finished with a room it always makes its way to the charging station. Then he seems to get the information about which room he has to clean next and then drives into it.
In the MAP it also looks as if they were individual cleaning orders, as only the active room is marked in color (and shown with the route already taken).

In the original app, all rooms and the complete route that the robot traveled are shown.

So my question is, am I using the wrong function?

I look forward to your answers.

**Comment by dirkhe** [comment, marked as answer]
Wenn du den einzelnen drückst, packt er den immer einzeln in die queue. Du kannst entweder über via sendCommand, custom command (siehe readme) oder über die Timerfunktion gehen. Da kannst du einen Timer anlegen, dort mehrere Räume zuweisen. Du könntest den dann auf inaktiv setzten und dann bei Bedarf auf jetzt reinigen einmalig ausführen
